packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/ManageTopicsSection/ManageTopicsSection.scss

```diff
@@ -15,13 +15,26 @@
   gap: 24px;
 
   .content {
-    .featured-topic-list {
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+
+    .featured-topic-list,
+    .regular-topic-list {
       display: flex;
       flex-direction: column;
-      height: 320px;
+      height: 400px;
```

Comment:
do we need this to be fixed height?

Reply:
The draggable requires it's parent container to have a specific height.